src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -160,7 +160,7 @@
             "courseId":"idOfTypicalCourse1.access",
             "name":"Helper Course1",
             "email":"helper@course1.tmt",
-            "role":"Helper",
+            "role":"Custom",
```

Comment:
Perhaps we can add some kind of validation logic when we load the instructorAttributes into the Backdoor to ensure that the privileges actually match the role. Otherwise, it will be too easy for developers in future to repeat the same error.